compiler/src/dotty/tools/dotc/config/Feature.scala

```diff
@@ -28,7 +28,19 @@ object Feature:
   val symbolLiterals = deprecated("symbolLiterals")
   val fewerBraces = experimental("fewerBraces")
 
```

Comment:
@odersky FYI, putting `fewerBraces` under `experimental` means that it will only available for nightly build of compilers after this PR. Normal releases cannot use the feature.

Reply:
Yes, that's understood.